src/test/java/api/InstancesApiExamples.java

```diff
@@ -826,6 +832,31 @@ public void cannotFindAnUnknownInstance()
     assertThat(getResponse.getStatusCode(), is(404));
   }
 
+  @Test
+  public void cannotUpdateHrid() throws Exception {
```

Comment:
What happens if the HRID is not specified in the update?

Reply:
The update will be rejected.

Reply:
Please can a test be added to demonstrate that a HRID cannot be removed (for both instance and item).